debezium-connector-db2/pom.xml

```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-connector-db2</artifactId>
+    <name>Debezium Connector for IBM db2 Server</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <!--
+          Specify the properties that will be used for setting up the integration tests' Docker container.
+          Note that the `dockerhost.ip` property is computed from the IP address of DOCKER_HOST, which will
+          work on all platforms. We'll set some of these as system properties during integration testing.
+      -->
+        <sqlserver.port>1433</sqlserver.port>
+        <sqlserver.user>sa</sqlserver.user>
+        <sqlserver.password>Password!</sqlserver.password>
+        <sqlserver.dbname>testDB</sqlserver.dbname>
+        <docker.filter>microsoft/mssql-server-linux:2017-CU9-GDR2</docker.filter>
```

Comment:
Copy and Paste glitch from SQL Server?

Reply:
Updated to DB2 in the latest push, as well as some integration tests.